libipl : Perform setup ref clock and clock test only on master processor

devenrao opened the pull request:
Tested:
Aug 03 06:07:54 p10bmc openpower-proc-control[7996]: Istep: set_ref_clock:
started
Aug 03 06:07:54 p10bmc openpower-proc-control[7996]: Running p10_setup_ref_clock
HWP on master processor 0

Aug 03 06:07:54 p10bmc openpower-proc-control[7996]: Istep: proc_clock_test:
started
Aug 03 06:07:54 p10bmc openpower-proc-control[7996]: Running p10_clock_test HWP
on master processor 0

Change-Id: Ib7930b5d6fc8f91a84fa35493c4bcbb6cd7071ce
Signed-off-by: Marri  Rao <devenrao@in.ibm.com>

devenrao committed:
libipl : Perform setup ref clock only on master processor

Perform setup ref clock only on master processor targets as
host is incharge of secondary processors

Tested:
Aug 03 06:07:54 p10bmc openpower-proc-control[7996]: Istep: set_ref_clock:
started
Aug 03 06:07:54 p10bmc openpower-proc-control[7996]: Running p10_setup_ref_clock
HWP on master processor 0

Change-Id: Ib7930b5d6fc8f91a84fa35493c4bcbb6cd7071ce
Signed-off-by: Marri  Rao <devenrao@in.ibm.com>

ojayanth commented:
Not able to open the Commits.
High level split the commit into two. and add why this is getting disabled now

devenrao committed:
libipl : perform clock test only on master processor

Perform clock test only on primary processors as host is
incharge of secondary processors.

Tested:
Aug 03 06:07:54 p10bmc openpower-proc-control[7996]: Istep: proc_clock_test:
started
Aug 03 06:07:54 p10bmc openpower-proc-control[7996]: Running p10_clock_test HWP
on master processor 0

Signed-off-by: Marri Devender Rao <devenrao@in.ibm.com>
Change-Id: I335fe0513c95537035ce22408df2044fdf0c0d08
